src/query/execute_schema.c

```diff
@@ -2774,47 +2757,49 @@ create_or_drop_index_helper (PARSER_CONTEXT * parser, const char *const constrai
       return ER_NOT_ALLOWED_ACCESS_TO_PARTITION;
     }
 
-  if (comment != NULL)
+  ctype = get_reverse_unique_index_type (is_reverse, is_unique);
+
+  char *attname_tmp = NULL;
+  if (do_index != DO_INDEX_CREATE)
     {
-      assert (comment->node_type == PT_VALUE);
-      comment_str = (char *) PT_VALUE_GET_BYTES (comment);
+      assert (constraint_name != NULL);
+      nnames = 0;
+      attnames = &attname_tmp;
+      attnames[0] = NULL;
```

Comment:
it looks not necessary. it is already NULL.

Reply:
It is put in such a way that developers who read the code are clearly aware that the names[0] should be NULL.
Excluding "nnames = 0;" and "attnames [0] = NULL;" is the same, but we want to keep it as it is for readability.